internal/scraper/scraper.go

```diff
@@ -314,12 +331,15 @@ func (s Scraper) collectData(ctx context.Context, t time.Time) (*probeData, erro
 		target = addCacheBustParam(s.target, s.check.Settings.Http.CacheBustingQueryParamName, s.probe.Name)
 	}
 
+	s.logger.Info().Msg("Collecting data")
 	// set up logger to capture check logs
 	logs := bytes.Buffer{}
 	bl := kitlog.NewLogfmtLogger(&logs)
 	sl := kitlog.With(bl, "ts", kitlog.DefaultTimestampUTC, "target", target)
 
-	success, mfs, err := getProbeMetrics(ctx, prober, target, s.bbeModule, s.buildCheckInfoLabels(), s.summaries, s.histograms, sl, s.check.BasicMetricsOnly)
+	success, mfs, err := getProbeMetrics(ctx, prober, target, s.bbeModule, s.buildCheckInfoLabels(), s.summaries, s.histograms, sl, s.check.BasicMetricsOnly, s.probe.Name)
```

Comment:
Shall we look into this more, as one of the first things we do? Maybe we can reverse this change, if the probe-name is already available to the node-graph / datasource somewhere/somehow?